static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -326,16 +326,21 @@ export function StacktraceLink({frame, event, line}: StacktraceLinkProps) {
               },
               {startSession: true}
             );
-            openModal(deps => (
-              <StacktraceLinkModal
-                onSubmit={handleSubmit}
-                filename={filename}
-                project={project}
-                organization={organization}
-                integrations={match.integrations}
-                {...deps}
-              />
-            ));
+            openModal(
+              deps => (
+                <StacktraceLinkModal
+                  onSubmit={handleSubmit}
+                  filename={filename}
+                  project={project}
+                  organization={organization}
+                  integrations={match.integrations}
+                  {...deps}
+                />
+              ),
+              {
+                backdrop: false,
```

Comment:
Did you mean to disable the backdrop?

Reply:
whoops was playing around with event stuff